app/ui_screen.js

```diff
@@ -153,11 +154,17 @@ const UI = {
                         },
                         false // Not a primary display
                     );
+        }
+        
+
         UI.rfb.addEventListener("connect", UI.connectFinished);
         //UI.rfb.addEventListener("disconnect", UI.disconnectFinished);
-        UI.rfb.clipViewport = UI.getSetting('view_clip');
-        UI.rfb.scaleViewport = UI.getSetting('resize', false, 'remote') === 'scale';
-        UI.rfb.resizeSession = UI.getSetting('resize', false, 'remote') === 'remote';
+        UI.rfb.forcedResolutionX = UI.getSetting('forced_resolution_x', false);
+        UI.rfb.forcedResolutionY = UI.getSetting('forced_resolution_y', false);
+        let resize_setting = UI.getSetting('resize', false, 'remote');
```

Comment:
As in ui.js, this should probably be a const, but if you prefer it as a let for some reason it's not a major thing.